src/sortStudents.ts

```diff
@@ -1,16 +1,54 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc'| 'desc';
 
+function averageGrade({ grades } : Student) :number {
+  return grades.reduce((sum, grade) => sum + grade, 0) / grades.length;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const sortedStudents = [...students];
+
+  return sortedStudents.sort((firstStudent, secondStudent) : number => {
```

Comment:
```suggestion
  return [...students].sort((firstStudent, secondStudent) : number => {
```
Unnecessary variable.